lib/l10n/intl_en.arb

```diff
@@ -165,6 +167,13 @@
   "filterMenuShowMine": "Show only mine",
   "filterMenuRelevance": "Filter by relevance",
 
+  "sourceOfficial": "Official web pages",
+  "sourceOfficialWebPages": "acs.pub.ro, upb.ro, curs.upb.ro, studenti.pub.ro",
+  "sourcesOrganization": "Student organization*",
+  "sourcesStudentOrganizations": "LSAC, BEST, MLSA",
+  "sourcesStudentRepresentative": "Representative students*",
```

Comment:
Add the stars from the code, not the localized strings. Here and everywhere else.